resources/lib/bossanova808/utilities.py

```
@@ -92,21 +98,24 @@ def get_setting(setting: str) -> str or None:
     return ADDON.getSetting(setting).strip()
```

Ruff (0.13.1)

`ADDON` may be undefined, or defined from star imports

(F405)